### ISSUE TITLE
main post input field misses keystrokes?

### ISSUE THREAD

**Issue by dmajoraddnine** [elsehow/hyperchat2#12]
If I type quickly on my mechanical keyboard in the top-level submit box, I lose characters.

This is the result of me typing the sentence "what up my doggies did you see that achewood is updating again" at ~70 wpm:

![chat-snip-1](https://cloud.githubusercontent.com/assets/1085461/13763169/9236c634-ea1a-11e5-8920-4c1d951adb13.PNG)

**Comment by dmajoraddnine** [comment]
Are you handling keyup inside the submit boxes?

**Comment by elsehow** [comment]
yeah, i am storing some local state. that's probably what's causing this. maybe keydown will be better? 

or, i should find another way of reading from the textbox (probably good old fashioned `document.getElementById`

**Comment by elsehow** [comment]
@dmajoraddnine can you confirm that 0f75ecbbd85829b64f256576114baa8b8e1eb2c9 fixes this?
